ods_ci/tests/Tests/100__deploy/100__installation/110__model_serving_watsonx.robot

```diff
@@ -15,8 +15,8 @@ ${KSERVE_NS}=    ${APPLICATIONS_NAMESPACE}
 *** Test Cases ***
 Verify KServe Is Shipped
     [Documentation]    Verify KServe Is Shipped And Enabled Within ODS
-    [Tags]    WatsonX
-    ...       ODS-2325
+    [Tags]    ODS-2325
```

Comment:
Missing TierX tag

Reply:
Optional: Usage of `Run Keyword And Continue On Failure` can be replaced by adding `[Tags]    robot:recursive-continue-on-failure`. This is documented here https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#continue-on-failure

Reply:
>Missing TierX tag

on purpose for this one. I'm not sure about keeping it because it requires the pre-req operators to be installed and it's not done in this suite. Will need to re-assess it's utility